experimental/framework/tinylicious-client/src/test/TinyliciousClient.spec.ts

```diff
@@ -5,25 +5,142 @@
 
 import { strict as assert } from "assert";
 import { v4 as uuid } from "uuid";
-import { SharedMap } from "@fluid-experimental/fluid-framework";
+import { DiceRoller } from "@fluid-example/diceroller";
+import { SharedMap, SharedDirectory } from "@fluid-experimental/fluid-framework";
 import { ContainerSchema } from "@fluid-experimental/fluid-static";
 import {
     TinyliciousClient,
     TinyliciousConnectionConfig,
     TinyliciousContainerConfig,
 } from "..";
 
-describe("TinyliciousClient", () => {
+describe("TinyliciousClient Pre-Initilization", () => {
+    let documentId: string;
+    beforeEach(() => {
+        documentId = uuid();
+    });
+
+    /**
+    * Scenario: test if TinyliciousClient can get a container without being initialized.
+    *
+    * Expected behavior: an error should be thrown when trying to get a container
+    * without initializing TinyliciousClient.
+    */
+    it("get a container without initializing", async () => {
```

Comment:
the test names here should be descriptive of the scenario (and ideally also form a natural language sentence with "it").  this test name (which is all you see in the test run output) suggests that you _can_ get a container without initializing. something like this would be more appropriate:
```
it("cannot get a container without initializing", ...)
```

Reply:
good point! Updated the test names.